lib/kvdb/ikvdb.c

```diff
@@ -372,12 +415,38 @@ ikvdb_create(const char *kvdb_home, struct kvdb_cparams *params)
     return err;
 }
 
+static merr_t
+ikvdb_is_pmem_only(const char *kvdb_home, const struct kvdb_meta *meta, bool *pmem_only)
+{
+    merr_t err;
+    int    i;
+
+    INVARIANT(kvdb_home);
+    INVARIANT(meta);
+    INVARIANT(pmem_only);
+
+    *pmem_only = false;
+
+    err = dax_path_is_fsdax(kvdb_home, pmem_only);
+    if (err) {
+         log_err("Failed to determinte the KVDB type for %s", kvdb_home);
```

Comment:
That message will confuse anyone who comes across it since KVDBs don't have types.  Perhaps the message should state: "Cannot determine if %s is on a DAX filesystem".  Maybe even proceed with the assumption that it is not a DAX filesystem?

Reply:
Fixed the message. Let me think about proceeding when this call fails.